Source/santad/EventProviders/SNTEndpointSecurityProcessFileAccessAuthorizer.mm

```diff
@@ -13,21 +13,29 @@
 /// limitations under the License.
 
 #import "Source/santad/EventProviders/SNTEndpointSecurityProcessFileAccessAuthorizer.h"
+#include "Source/santad/DataLayer/WatchItemPolicy.h"
+#include "Source/santad/EventProviders/SNTEndpointSecurityEventHandler.h"
 
+using santa::IterateProcessPoliciesBlock;
 using santa::Message;
+using santa::ProcessWatchItemPolicy;
 
 @interface SNTEndpointSecurityProcessFileAccessAuthorizer ()
 @property bool isSubscribed;
+@property IterateProcessPoliciesBlock iterateProcessPoliciesBlock;
```

Comment:
Make this a `(copy)`, and as you're not using the setter in init, you'll need to manually copy there.